### PR TITLE
data: theleocodes

### PR DESCRIPTION
**Pull request by theleocodes**
Added my data in data directory.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2918"></a>

### CODE REVIEW

File: data/theleocodes.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ],
+    "links": [
+        {
+          "name": "Follow me on GitHub",
+          "url": "https://github.com/adityassharma-ss",
+          "icon": "FaGithub"
+        },
+        {
+          "name": "Follow me on Twitter",
+          "url": "https://twitter.com/",
```
Comment (ChinmayMhatre) [inline data/theleocodes.json:19]: You'll have to add your twitter username at the end of this url

File: data/theleocodes.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "bio": "**MERN-Stack Developer | Cloud-Computing",
+    "tags": ["ReactJS", "JavaScript", "Cloud", "MongoDB", "NodeJS"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/" },
+      { "icon": "FaGithub", "url": "https://github.com/theleocodes" }
+    ],
+    "links": [
+        {
+          "name": "Follow me on GitHub",
+          "url": "https://github.com/adityassharma-ss",
```
Comment (ChinmayMhatre) [inline data/theleocodes.json:14]: you'll have to update the github username in the url

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Hey, I noticed somethings you might want to change in the commit

File: data/theleocodes.json (hunk 2 above)
Comment (eddiejaoude) [reply data/theleocodes.json:14]: Good spot @ChinmayMhatre 👍

File: data/theleocodes.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Tanvi Saini",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**MERN-Stack Developer | Cloud-Computing",
+    "tags": ["ReactJS", "JavaScript", "Cloud", "MongoDB", "NodeJS"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/" },
```
Comment (EmmaDawsonDev) [inline data/theleocodes.json:8]: Do you want this to link to your twitter account? You need to add your twitter username at the end

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just a few things to update in this PR before it can be merged. Let us know if you need any help with that.

**Comment by SaraJaoude** [comment]
Hi @theleocodes 

We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.eddiehub.io/docs/quickstart

Editing your Profile: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request closed without merging**